documentation/asciidoc/accessories/camera/lens.adoc

```diff
@@ -16,12 +16,9 @@ We recommend two lenses, a 6mm wide angle lens and a 16mm telephoto lens. These
 2+| Image format | 1" | 1/2"
 2+| Aperture | F1.4 to 1.6 | F1.2
```

Comment:
```suggestion
2+| Aperture | F1.4 to F16 | F1.2
```
Suggestion: update this row based on the recommendations in https://github.com/raspberrypi/documentation/issues/3255